packages/react-intl/src/formatters/message.ts

```diff
@@ -72,6 +73,29 @@ function deepMergeFormatsAndSetTimeZone(
   };
 }
 
+export function patchFormatXMLElementFn(
+  values: Record<
+    string,
+    | PrimitiveType
+    | React.ReactNode
+    | FormatXMLElementFn<React.ReactNode, React.ReactNode>
+  >
+): typeof values {
+  return Object.keys(values).reduce((acc: typeof values, k) => {
```

Comment:
Not that I am worried, but another breaking change to note here is that non-enumerable properties and non-own properties won't be interpolated by the formatter anymore.

Let's make a note in the changelog.

Reply:
It could be refactored to use `for ... in` loop to preserve all enumerable properties including non-own, but I don't see any real use-case for such support. Notice in changelog and docs shall be sufficient. But if you like, I can use `for ... in`.

Reply:
Making a note in the changelog sounds good to me.